otter/test/integration/lib.py

```diff
@@ -0,0 +1,53 @@
+"""The lib module provides a library of classes and functions useful for
+writing integration tests in the context of the Otter project.
+"""
+
+
+class IdentityV2(object):
+    """This class provides a way to configure commonly used parameters
+    exactly once for any number of Identity-related API calls.
+    """
+
+    def __init__(self, auth, username, password, endpoint, pool=None):
```

Comment:
Using [`attributes`](https://characteristic.readthedocs.org/en/stable/api.html#characteristic.attributes) decorator can help removing some of the boilerplate code. You can see examples [here](https://github.com/rackerlabs/otter/blob/master/otter/convergence/model.py).